core/src/main/java/pl/agh/edu/language/LanguageManager.java

```diff
@@ -28,6 +28,10 @@ public static String get(String key) {
 		return bundle.get(key);
 	}
 
+	public static String bound(String key, Object... varArgs) {
```

Comment:
jak co paweł stworzył klasę do zamieniania wartości w stringach i wykorzystywał ją w backendzie ja w HS-146 zrobiłem do tego adapter, więc jak chcesz możesz przerobić to na moją wersję albo zamienić tam wszystko co było wcześniej

Reply:
chociaż w case do {0} $ możemy po prostu dodać tego dolara bo dla każdego języku jest to samo

Reply:
a jak co to paweł też tam wcześniej używał formy {{ }} a nie { }